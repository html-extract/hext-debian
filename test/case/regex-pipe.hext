<a href:filter(/-([0-9]+)$/):user_id
   @text:trim:user_name
/>
