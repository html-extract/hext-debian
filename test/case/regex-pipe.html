<div>
  <a href="/user/bob-22"> Bob </a>
  <a href="/user/alice-23"> Alice </a>
  <a href="/user/tom-24"> Tom </a>
</div>
