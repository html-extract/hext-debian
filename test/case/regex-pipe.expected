{"user_id":"22","user_name":"Bob"}
{"user_id":"23","user_name":"Alice"}
{"user_id":"24","user_name":"Tom"}
